### PR TITLE
[BZ1982733]:Add redhat.registry.connect.com to the table

### PR DESCRIPTION
**Pull request by xenolinux**
OCP version: 4.6+
[Preview](https://deploy-preview-41761--osdocs.netlify.app/openshift-enterprise/latest/installing/install_config/configuring-firewall.html)
[Bug](https://bugzilla.redhat.com/show_bug.cgi?id=1982733)
QE contact: @yunjiang29

### CODE REVIEW

File: modules/configuring-firewall.adoc [hunk 1]
```diff
@@ -38,6 +38,10 @@ There are no special configuration considerations for services running on only c
 |443, 80
 |Provides {op-system-first} images
 
+|`registry.connect.redhat.com`
```
Comment (yunjiang29) [inline modules/configuring-firewall.adoc:41]: here should be `rhc4tp-prod-z8cxf-image-registry-us-east-1-evenkyleffocxqvofrk.s3.dualstack.us-east-1.amazonaws.com`

**Comment by yunjiang29** [comment]
@xenolinux And I also think it should be added in table `Allowlist the following URLs for optional third-party content` https://bugzilla.redhat.com/show_bug.cgi?id=1982733#c7

File: modules/configuring-firewall.adoc (hunk 1 above)
Comment (xenolinux) [reply modules/configuring-firewall.adoc]: Addressed

**Comment by xenolinux** [comment]
> @xenolinux And I also think it should be added in table `Allowlist the following URLs for optional third-party content` https://bugzilla.redhat.com/show_bug.cgi?id=1982733#c7

Addressed

File: modules/configuring-firewall.adoc [hunk 2]
```diff
@@ -38,6 +38,10 @@ There are no special configuration considerations for services running on only c
 |443, 80
 |Provides {op-system-first} images
 
+|`rhc4tp-prod-z8cxf-image-registry-us-east-1-evenkyleffocxqvofrk.s3.dualstack.us-east-1.amazonaws.com`
+|443
+|Provides access to container images hosted on `registry.connect.redhat.com`
+
```
Comment (yunjiang29) [inline modules/configuring-firewall.adoc:44]: revert this section as it already represents in `Allowlist the following URLs for optional third-party content`

File: modules/configuring-firewall.adoc [hunk 3]
```diff
@@ -165,6 +169,10 @@ that is specified in the `spec.route.hostname` field of the
 |`oso-rhc4tp-docker-registry.s3-us-west-2.amazonaws.com`
 |443, 80
 |Required for Sonatype Nexus, F5 Big IP operators.
+
+|`rhc4tp-prod-z8cxf-image-registry-us-east-1-evenkyleffocxqvofrk.s3.dualstack.us-east-1.amazonaws.com`
+|443
```
Comment (yunjiang29) [inline modules/configuring-firewall.adoc:174]: 443, 80

File: modules/configuring-firewall.adoc [hunk 4]
```diff
@@ -165,6 +169,10 @@ that is specified in the `spec.route.hostname` field of the
 |`oso-rhc4tp-docker-registry.s3-us-west-2.amazonaws.com`
 |443, 80
 |Required for Sonatype Nexus, F5 Big IP operators.
+
+|`rhc4tp-prod-z8cxf-image-registry-us-east-1-evenkyleffocxqvofrk.s3.dualstack.us-east-1.amazonaws.com`
```
Comment (yunjiang29) [inline modules/configuring-firewall.adoc:173]: and I suggest moving this row up one line, close to the `registry.connect.redhat.com`

File: modules/configuring-firewall.adoc (hunk 2 above)
Comment (xenolinux) [reply modules/configuring-firewall.adoc]: Addressed

File: modules/configuring-firewall.adoc (hunk 3 above)
Comment (xenolinux) [reply modules/configuring-firewall.adoc]: Addressed

File: modules/configuring-firewall.adoc (hunk 4 above)
Comment (xenolinux) [reply modules/configuring-firewall.adoc]: Addressed

**Review by EricPonvelle — APPROVED** [review]
/lgtm

**Pull request merged**